### ISSUE TITLE
[Suggestion] Issues regarding to refresh AltStore by WiFi connection

### ISSUE THREAD

**Issue by MobiusMei** [rileytestut/AltStore#904]
So I have been using AltStore for a while, which works great on both my iPad and my iPhone. 
The annoying problem was, I could refresh the 7-days access limit for my iPad by WiFi connection, while the iPhone must be plugged-in for this with the same WiFi.
However, I realized that it's not working perfectly on my iPhone simply because I trapped myself. 

It's a common, easy mistake that even the FAQ page is not listing this. However, considering people like me, I believe that we should list the following hint under the FAQ page, under the section [AltStore says “Could not find AltServer” when trying to refresh].

Long story short, on your device, go to Settings >> (Scroll all the way down to apps) AltStore, and check if "Local Network" permission is granted. Without this, the wireless refresh wouldn't work.

Anyway, shame on myself for failing to locate such obvious mistake for like 3-4 weeks, and not noticed at all.
